applications/FluidDynamicsApplication/custom_strategies/convergence_criteria/vel_pr_criteria.h

```diff
@@ -169,14 +169,20 @@ class VelPrCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpace >
             TDataType VelAbs = sqrt(VelIncreaseNorm)/ static_cast<TDataType>(VelDofNum);
             TDataType PrAbs = sqrt(PrIncreaseNorm)/ static_cast<TDataType>(PrDofNum);
 
-            std::cout << "CONVERGENCE CHECK:" << std::endl;
-            std::cout << " VELOC.: ratio = " << VelRatio <<"; exp.ratio = " << mVelRatioTolerance << " abs = " << VelAbs << " exp.abs = " << mVelAbsTolerance << std::endl;
-            std::cout << " PRESS.: ratio = " << PrRatio <<"; exp.ratio = " << mPrRatioTolerance << " abs = " << PrAbs << " exp.abs = " << mPrAbsTolerance << std::endl;
+            if (rModelPart.GetCommunicator().MyPID() == 0 && this->GetEchoLevel() > 0)
+            {
```

Comment:
Just so you know, this convergence criteria is never used in MPI, so the communicator stuff is not strictly required (there is a separate velocity/pressure criteria in the trilinos application).
This is correct, so no need to fix it here.